styles.css

```diff
@@ -0,0 +1,172 @@
+* {
+	box-sizing: border-box;
+}
+
+body {
+	font-family: arial, sans-serif;
+	
+}
+
+nav {
+	background-color: #019dcf;
+	color: #ffffff;
+	font-size: 20px;
+	font-weight: 400;
+	letter-spacing: .05em;
+	padding: 40px 60px;
+}
+
+nav ul {
+	margin: 0;
+	width: 100%;
+	padding: 0;
+	text-align: center;
+	overflow: auto;
+}
+
+nav li {
+	float: left;
+	list-style: none;
+	width: 33%;
+}
+
+nav li:hover {
+	color: black;
+}	
+
+li a {
+	color: #ffffff;
+}
+
+header {
+	background-color: #ffffff;
+	color: #019dcf;
+	font-size: 40px;
+	font-weight: 400;
+	letter-spacing: .05em;
+	padding: 10px 20px;
+	text-align: center;
+
+}
+
+.portfolio {
+	margin: 20px 20px;
+	overflow: auto;
+}
+
+.portfolio .project {
+	background-color: #019dcf;
+	color: #ffffff;
+	font-size: 20px;
+	font-weight: 400;
+	letter-spacing: .05em;
+	padding: 40px 60px;
+	width: 33%;
+	float: left;
+	background-repeat: no-repeat;
+	background-size: auto;
+	background-position: center;
+}
+
+
+.portfolio #projectOne {
+	background-image: url("https://cdn.pixabay.com/photo/2019/08/29/19/48/cobweb-4439844__340.jpg");
```

Comment:
save images locally so you dont have to rely on that website being live for ever